src/Scaffold/Console/controller/controller.stub

```diff
@@ -16,6 +16,9 @@ class {{studly_name}} extends Controller
     public $formConfig = 'config_form.yaml';
     public $listConfig = 'config_list.yaml';
 
+    {%~ if layout == 'sidebar' %}
```

Comment:
@alxy I am not familiar with this twig syntax, what does "{%~" do exactly?

Reply:
@mjauvin I believe it removes all whitespace before the tag.

Reply:
@bennothommo I think it should be "{%-" (dash), no?

Reply:
@mjauvin Indeed you're right. Serves me right for trying to sound smart at 11pm. :P

Reply:
I tried around a bit and ~ does work very similar to - but is a bit different: https://symfony.com/blog/better-white-space-control-in-twig-templates
The solution is not yet optimal, I was not able to get the correct amount of new lines for both cases (default and sidebar)

Reply:
The current code produces the following output.

Layout default:
```
public $formConfig = 'config_form.yaml';
public $listConfig = 'config_list.yaml';


public function __construct()
```

Layout sidebar:
```
public $formConfig = 'config_form.yaml';
public $listConfig = 'config_list.yaml';

public $bodyClass = 'compact-container';
    
public function __construct()
```

The default layout has one additional newline which is unnecesary here.

Reply:
> I tried around a bit and ~ does work very similar to - but is a bit different: https://symfony.com/blog/better-white-space-control-in-twig-templates

Oh, nice! Have been looking for this one a year ago and didn't find it.

Thanks!